src/main/groovy/com/marklogic/gradle/task/roxy/CopyRoxyFilesTask.groovy

```diff
@@ -0,0 +1,53 @@
+package com.marklogic.gradle.task.roxy
+
+import com.marklogic.gradle.task.MarkLogicTask
+import org.apache.commons.io.FileUtils
+import org.gradle.api.tasks.TaskAction
+
+import static groovy.io.FileType.FILES
+
+class CopyRoxyFilesTask extends MarkLogicTask {
+
+	def roxyFolderMapping = [
+								"src" : "src/main/ml-modules/root",
+								"rest-api/config" : "src/main/ml-modules/options",
+								"rest-api/ext" : "src/main/ml-modules/services",
+								"rest-api/transforms" : "src/main/ml-modules/transforms"
+							]
+
+	@TaskAction
+	void copyRoxyFiles() {
+		backupProperties()
+		if (getRoxyHome()) {
+			copyFolders()
+		} else {
+			println "mlRoxyHome parameter is not provided. Please run using -PmlRoxyHome=/your/roxy/project/home"
+		}
+	}
+
+	void copyFolders() {
+		roxyFolderMapping.each { k, v ->
+			def source = getRoxyHome() + "/" + k
+			def sourceFolder = new File(source)
+			if (sourceFolder.exists() && sourceFolder.isDirectory()) {
+				println "Creating folder '" + v + "' ... "
+				def targetFolder = new File(v)
+				FileUtils.forceMkdir(targetFolder)
+				println "Copying contents of '" + source + "' to '" + v + "' ... "
+				FileUtils.copyDirectory(sourceFolder, targetFolder)
+			}
+		}
+	}
+
+	void backupProperties() {
```

Comment:
Looks like you can remove this, because you're not making any modifications to gradle.properties.